package.json

```diff
@@ -22,8 +23,9 @@
     "url": "https://github.com/miragejs/graphql/issues"
   },
   "scripts": {
+    "build": "node build && tsc",
```

Comment:
Is there a reason for adding a `build` script here vs. having just the pre-publish script for npm?

Reply:
In most projects I've worked on the `build` command was used to build the project. It just felt weird to run `npm run prepublishOnly` to build the project whilst configuring TypeScript. 

I can revert that if needed.

Reply:
Makes sense but I'm good with duplicating `node build && tsc` for each script (`build` and `prepublishOnly`), for now. I mostly want to get the `npm run build` out of there since we're using Yarn in the project.